dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/controller/GaenController.java

```diff
@@ -224,7 +227,8 @@ public GaenController(
 
     // Filter out non valid keys and insert them into the database (c.f. InsertManager and
     // configured Filters in the WSBaseConfig)
-    insertIntoDatabaseIfJWTIsNotFake(gaenSecondDay.getDelayedKey(), userAgent, principal, now);
+    insertExposedNextDayIntoDatabaseIfJWTIsNotFake(
+        gaenSecondDay.getDelayedKey(), userAgent, principal, now);
```

Comment:
```suggestion
    insertManagerExposedNextDay.insertIntoDatabase(keys, userAgent, principal, now);
```

Reply:
we need to make a list out of the single key. But I will shortcut this with List.of()